app/ui-react/syndesis/src/modules/integrations/components/editor/step/WithConfigurationForm.tsx

```diff
@@ -108,7 +108,7 @@ export const WithConfigurationForm: React.FunctionComponent<
     definition = step.properties;
   }
 
-  const initialValue = step.configuredProperties;
+  const initialValue = props.step.configuredProperties;
```

Comment:
🤦‍♂️ sorry

Reply:
I knew you'd hate to see that, no worries though!  at least it was something simple :-)